model/src/main/java/org/jboss/pnc/rex/model/requests/StopRequest.java

```diff
@@ -31,7 +32,7 @@
 @ToString
 public class StopRequest {
 
-    private final String callback;
+    private final Request callback;
```

Comment:
can you specify the full java package here to address the probable confusion between Request classes

Reply:
Yup will do!